### PR TITLE
fix(checkout): CHECKOUT-7328 Update discounted_amount description

### PR DESCRIPTION
**Pull request by valentindellangela**
# [[DEVDOCS-]](https://bigcommercecloud.atlassian.net/browse/CHECKOUT-7328)

## What changed?
* description of the field `discounted_amount` in our Cart and Checkout API has been updated to be clearer as it's been raised it was confusing this field was returning 0 when adding a coupon code.

From `Discounted amount.` to `Order based discounted amount only - Coupon discounts and product based discounts are excluded.`

## Anything else?
N/A

cc @bigcommerce/checkout  @bigcommerce/dev-docs-collaborators

### CODE REVIEW

**Comment by valentindellangela** [comment]
Oups sorry I missed this and thought maybe I forgot to open it for review
image

File: reference/carts.v3.yml [hunk 1]
```diff
@@ -1324,7 +1324,7 @@ components:
           description: 'Sum of cart line-item amounts before cart-level discounts, coupons, or taxes.'
         discount_amount:
           type: number
-          description: Discounted amount.
+          description: Order based discounted amount only - Coupon discounts and product based discounts are excluded.
```
Comment (bc-traciporter) [inline reference/carts.v3.yml:1327]:
```suggestion
          description: Order-based discounted amount only - Excludes coupon discounts and product-based discounts.
```

File: reference/checkouts.sf.yml [hunk 2]
```diff
@@ -2706,7 +2706,7 @@ components:
                   format: double
                 discountAmount:
                   type: number
-                  description: Discounted amount.
+                  description: Order based discounted amount only - Coupon discounts and product based discounts are excluded.
```
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:2709]:
```suggestion
                  description: Order-based discounted amount only - Excludes coupon discounts and product-based discounts.
```

File: reference/checkouts.v3.yml [hunk 3]
```diff
@@ -7846,7 +7846,7 @@ components:
               description: ID of channel
             discount_amount:
               type: number
-              description: Discounted amount.
+              description: Order based discounted amount only - Coupon discounts and product based discounts are excluded.
```
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:7849]:
```suggestion
              description: Order-based discounted amount only - Excludes coupon discounts and product-based discounts.
```

**Review by bc-traciporter — COMMENTED** [review]
Left some editorial comments

**Comment by valentindellangela** [comment]
Thanks @bc-traciporter  - I've updated the PR based on your feedback.

**Pull request merged**